.github/workflows/main.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+    - uses: actions/checkout@master
+    - name: Install Rust (rustup)
+      run: rustup update stable --no-self-update && rustup default stable
```

Comment:
Github Actions has rust installed by default?

Reply:
It does on Ubuntu/Windows, yes, but this is done to ensure it's the latest stable. Typically it's a noop that finishes immediately, but should help insulate against a period where there's maybe not updates for awhile.